taskcluster/win-build.sh

```diff
@@ -3,6 +3,11 @@
 set -xe
 
 cuda=$1
+runtime=$1
+
+if [ "${cuda}" != "--cuda" ]; then
+    cuda=
+fi;
```

Comment:
Nice hack, but it took me way too much time to understand for what should be a simple script, please separate these into different parameters or make downstream consumers understand `--cuda` vs `--tflite`.

Reply:
Updated downstream to handle properly